src/mlpack/tests/main_tests/pca_test.cpp

```diff
@@ -48,7 +52,7 @@ struct PCATestFixture
   PCATestFixture()
   {
     // Cache in the options for this program.
-    CLI::RestoreSettings(mlpack::bindings::tests::programName);
+    CLI::RestoreSettings(pcaProgramName);
```

Comment:
This could be replaced with `CLI::RestoreSettings(PROGRAM_NAME)` to avoid the static variable.